commands/staff/am.js

```diff
@@ -1,20 +1,19 @@
 const { fn, emojis } = require("../../config")
-const db = require("quick.db")
-const { players } = require("../../db.js")
+// const { players } = require("../../db.js")
 
 module.exports = {
     name: "am",
     aliases: ["addmoney"],
     staffOnly: true,
-    run: async (message, args, client) => {
+    run: async (message, args) => {
         let amount = parseInt(args[0])
         if (!amount) return message.channel.send(`${args[0]} is not a valid amount`)
         args.shift()
         msg = ``
         args.forEach(async (x) => {
             let user = fn.getUser(x, message)
             if (user) {
-                let data = players.findOneAndUpdate({ user: user.id }, { $inc: { coins: amount } }).exec()
+                // let data = players.findOneAndUpdate({ user: user.id }, { $inc: { coins: amount } }).exec()
```

Comment:
Um... why would you comment out the line that actually adds the coins to the database??

Reply:
077636f